### PR TITLE
Update swag shop to be compatible with Reaction v1.10.0

### PR DESCRIPTION
**Pull request by prinzdezibel**

### CODE REVIEW

**Comment by prinzdezibel** [comment]
@mori5ive  Swag shop should work with release-1.9.0. It still needs to be reviewed, but in case you're in hurry you may checkout the PR: #52

File: client/components/product-variant/customer/productGrid.js [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
+  }
+
+  constructor(...args) {
+    super(...args);
+    this.tagRouteSlug = Reaction.Router.getParam("slug");
+  }
 
-class Products extends ProductsCore {
   heroClicked = () => {
     Logger.info("clicked");
```
Comment (zenweasel) [inline client/components/product-variant/customer/productGrid.js]: Should we remove this?

**Comment by zenweasel** [comment]
I'm getting this error on startup:
```
TypeError: Cannot read property '_id' of undefined
    at Object.methods.publishProducts (imports/plugins/custom/reaction-swag-shop/server/methods.js:101:20)
    at Hooks.Events.add (imports/plugins/custom/reaction-swag-shop/server/init.js:17:11)
    at callbacks.reduce (server/api/hooks.js:62:51)
    at Array.reduce (<anonymous>)
```

In devtools I am just importing that function directly rather than going all of that to run the Meteor method.

File: package.json [hunk 2]
```diff
@@ -54,5 +58,18 @@
       "react",
       "stage-2"
     ]
+  },
+  "eslintConfig": {
+    "extends": "@reactioncommerce"
+  },
+  "devDependencies": {
+    "eslint-import-resolver-meteor": "^0.4.0",
```
[the remark this answers is not in the document]
Comment (prinzdezibel) [reply package.json]: I needed that for running the lint script standalone in the plugin directory.

File: client/components/product-variant/customer/productGrid.js (hunk 1 above)
Comment (prinzdezibel) [reply client/components/product-variant/customer/productGrid.js]: The heroClicked handler that is?

**Pull request merged**